mod/01_core/set/all/content.rb

```diff
@@ -19,8 +19,11 @@ def raw_content
   structure ? template.db_content : db_content
 end
 
-def chunk_list #override to customize by set
-  :default
+
+format :all do
+  def chunk_list #override to customize by set
+    :default
+  end
```

Comment:
I thought there is a `chunk_list` defined for html format but there isn't. It's on the html cardtype set.
So I agree. We don't need it for chunk_list. At least not at the moment.
But I think in general it's useful. By default format beats set. Whenever you need it the other way you need `format(:all)`.
For example for css cards we don't want to render urls in any format. So we define
`Set::Type::Css::Format.chunk_list`. We are lucky because there is no `chunk_list` defined on a broader set in any format. A `Set::All::HtmlFormat.chunk_list` would mess up the html format of css cards.

@ethn: You are still acting as 'wagn'. Maybe you forgot to transform back to 'ethn' after I asked you to use your super powers ;)